app/controllers/ProofOfEntitlementController.scala

```diff
@@ -41,14 +48,19 @@ class ProofOfEntitlementController @Inject() (
     env:               Environment,
     ec:                ExecutionContext,
     cc:                MessagesControllerComponents,
-    frontendAppConfig: FrontendAppConfig
+    frontendAppConfig: FrontendAppConfig,
+    auditor:           AuditorService
 ) extends ChildBenefitBaseController(authConnector) {
   val view: Action[AnyContent] =
     Action.async { implicit request =>
-      authorisedAsChildBenefitUser { _ =>
+      authorisedAsChildBenefitUser { authContext =>
         childBenefitEntitlementConnector.getChildBenefitEntitlement.fold(
           err => errorHandler.handleError(err),
-          entitlement => Ok(proofOfEntitlement(entitlement))
+          entitlement => {
+            fireAuditEvent(request, entitlement, authContext)
```

Comment:
```suggestion
            auditor.viewProofOfEntitlement(authContext.nino, status = "Success", request, entitlement.some)
```

You also need to handle account not found